src/listpack.c

```diff
@@ -218,7 +218,7 @@ int lpStringToInt64(const char *s, unsigned long slen, int64_t *value) {
 /* Create a new, empty listpack.
  * On success the new listpack is returned, otherwise an error is returned.
  * Pre-allocate at least `capacity` bytes of memory,
- * over-allocated memory can be shrinked by `lpShrinkToFit`.
+ * over-allocated memory can be sharnk by `lpShrinkToFit`.
```

Comment:
you mean the spell checker didn't detect the old typo or the new ones?
or maybe you mean that you didn't yet run the spell checker on the modified one?

Reply:
It detect the old one. But did not detect the new one.
Like a map...
https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary.txt#L25690

Reply:
i don't understand... does it only detect common typos, and ignores typos it doesn't know?
does it match the test to a whitelist dictionary of good words? or against a blacklist dictionary of known typos?
(sorry, i'm too lazy / busy to check myself)

Reply:
ok, i see it uses a blacklist. i.e. it can only detect common typos, but will not detect uncommon ones.
i.e. the word "encyclop**a**dia" would pass, since the only common typo it can detect in `encyclopedia` is when the second `c` is missing.
well, i think this is very poor. but maybe it's better than warning about domain-specific terms and variable names.
i guess "something is better than nothing"